tests/rkt_api_service_test.go

```diff
@@ -230,7 +271,40 @@ func TestAPIServiceListInspectPods(t *testing.T) {
 		t.Errorf("Unexpected result: %v, should see zero pods", resp.Pods)
 	}
 
-	patchImportAndRun("rkt-inspect-print.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}, t, ctx)
+	patches := []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}
+	patchImportAndFetchHash("rkt-inspect-print.aci", patches, t, ctx)
+	// TODO(derekparker) There is a bug where `patchImportAndFetchHash` does not
```

Comment:
cc @jonboulle We need `rkt fetch` to be able to take a flag and return a full hash.